2-mandatory/2-water-bottle.js

```diff
@@ -15,22 +15,26 @@ You have to implement the missing features according to the specification.
 */
 
 // Here is your starting point:
-let bottle = {
+const UNITS = 10;
+const bottle = {
   volume: 0,
-  fillUp: function () {
-    // calling this function should completely fill your bottle (volume = 100);
+  capacity: 100,
+  fillUp() {
+    this.volume = 100;
```

Comment:
One more magic number to be removed :) 

```suggestion
    this.volume = this.capacity;
```